frontend/src/metabase/querying/filters/components/QuarterYearPicker/QuarterYearPicker.tsx

```diff
@@ -0,0 +1,30 @@
+import dayjs from "dayjs";
+
+import type { QuarterYearPickerValue } from "metabase/querying/filters/types";
+import { Box, QuarterPicker } from "metabase/ui";
+
+type QuarterYearPickerProps = {
+  value?: QuarterYearPickerValue;
```

Comment:
Let's indicate it's expected that this prop is passed

```suggestion
  value: QuarterYearPickerValue | undefined;
```